assets/i18n/en_US.json

```diff
@@ -220,7 +222,9 @@
 
         "deleteKeystoreLabel": "Delete keystore",
         "deleteKeystoreHint": "Delete the keystore used to sign the app",
-        "deleteKeystoreDialogText": "Are you sure you want to delete the keystore used to sign patched applications?",
+
+        "deleteKeystoreDialogTitle": "Delete keystore?",
```

Comment:
Does deleting this keystore cause the Manager to generate a new one? If so, do `Regenerate keystore?` instead or something in similar terms.